CONTRIBUTING.md

```diff
@@ -107,6 +108,12 @@ You can also start Jest in "watch" mode:
 npm test -- --watchAll
 ```
 
+We also prepared a helper script that runs tests only for changed files:
+
+```shell
+npm run test-watch
```

Comment:
Shouldn't this be `npm run test-changed`?

Reply:
Of course (the day was too long, sorry... 😅) I'll update my PR immediately.